index.html

```diff
@@ -11,98 +11,130 @@
-                                fill="#505F79"
-                            />
-                        </svg>
+                <div class="headline__link-container">
+                    <a href="#works" class="headline__link">
+                        <span class="material-symbols-outlined"> expand_more </span>
                     </a>
-                </li>
-            </ul>
-            <div class="headline__link-container">
-                <a href="#works" class="headline__link">
-                    <span class="material-symbols-outlined"> expand_more </span>
-                </a>
-            </div>
-        </section>
+                </div>
+            </section>
+        </header>
         <main>
             <section class="works" id="works">
```

Comment:
- [ Optional ] Please consider adding more placeholder projects to your portfolio, this will make it easier for you to add new projects in the future to your portfolio. In Microverse you'll work on more projects in the months to come, thus it would be better if you added more placeholders to your portfolio as shown in the Figma template below.

## Figma template
![Screenshot from 2022-12-08 18-34-21](https://user-images.githubusercontent.com/16392046/206488234-379ad2ab-a3d9-4af1-8ae2-29c570cf1a35.png)

## Your Implementation of the design
![Screenshot from 2022-12-08 18-36-17](https://user-images.githubusercontent.com/16392046/206488858-66384e8f-d2a8-491c-af93-76a89ee470e9.png)